api/nakadi-event-bus-api.yaml

```diff
@@ -799,6 +799,112 @@ paths:
             $ref: '#/definitions/Problem'
 
 
+  /subscriptions/{subscription_id}/cursors:
+    get:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Exposes the currently committed offsets of a subscription.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
```

Comment:
The result that is retrieved here is not usable at all. We will have cursors like that 
```
[{'partition': '0', offset: 120}, {'partition': '1', offset: 100}]
```
It doesn't point to anywhere. My point is that we need to think about it and to include event type to cursor (or extended cursor) right now.